js/common/lib/inference-session.ts

```diff
@@ -200,6 +201,11 @@ export declare namespace InferenceSession {
   export interface XnnpackExecutionProviderOption extends ExecutionProviderOption {
     readonly name: 'xnnpack';
   }
+  export interface WebNNExecutionProviderOption extends ExecutionProviderOption {
+    readonly name: 'webnn';
+    deviceType?: number;       // 0 - auto, 1 - gpu, 2 - cpu
+    powerPreference?: number;  // 0 - auto, 1 - high-performance, 2 - low-power
```

Comment:
use string literals as they are more self-explanatory

Reply:
Let's keep sync with the spec: https://www.w3.org/TR/webnn/#api-ml

Reply:
Thanks, I agree👍